docs/make.jl

```diff
@@ -68,7 +68,8 @@ files = [
     # Topic: other stuff
     "Explicit time stepping" => "time_stepping.jl",
     "Differentiable programming" => "differentiable_programming.jl",
-    "Custom semidiscretizations" => "custom_semidiscretization.jl"
+    "Custom semidiscretizations" => "custom_semidiscretization.jl",
+    "Core aspects of the basic setup" => "innards_of_the_basic_setup.jl",
```

Comment:
```suggestion
    "Behind the scenes of a simulation setup"
     => "behind_the_scenes_simulation_setup.jl",
```
If the title is changed as proposed above, you should change also the filename to something like this (or something better 😬)